packages/eslint-plugin-lit-a11y/lib/utils/isCustomElement.js

```diff
@@ -0,0 +1,13 @@
+const CE_TAGNAME_RE = /^[a-z]\w+-\w+/;
+
+/**
+ * Is the element a custom element (i.e. does it contain a '-')?
+ * @param {import('parse5-htmlparser2-tree-adapter').Element} element
+ */
+function isCustomElement(element) {
+  return !!(element && typeof element.name === 'string' && CE_TAGNAME_RE.test(element.name));
```

Comment:
is there a case where `element.name` is not a string? o.O

Reply:
i have trauma with parse5 types.

element could be a node? should be guaranteed by tsc, but see previous sentence

Reply:
ugh types, alright, whatever then 🤷‍♂️